CHANGES.md

```diff
@@ -2,6 +2,7 @@ Airbase 128
 
 * Plugin updates:
   - JaCoCo 0.8.8 (from 0.8.7)
+* Ensure ``argLine`` is always defined, with or without Jacoco being enabled
```

Comment:
Or maybe 

```
* Always define ``argLine`` regardless whether Jacoco is enabled to help IntelliJ IDEA
  tests configuration from Surefire maven plugin's configuration.
```

Reply:
I feel like we're deep into bikeshed territory here ;) I'm kind of reluctant to mention IDEA specifically here, but not exactly sure why. But I don't have strong feelings about it so why not.

Reply:
How about now?

* Always define ``argLine`` property even if Jacoco is disabled, to ensure
  that Surefire configuration does not confuse IntelliJ IDEA